components/user_settings/security/user_access_token_section/user_access_token_section.jsx

```diff
@@ -49,6 +49,7 @@ export default class UserAccessTokenSection extends React.Component {
         super(props);
 
         this.state = {
+            active: null,
```

Comment:
Since this represents the current value of `this.props.active`, can you change this to default to the value from the props? That also makes it so that we don't mix types here since that can cause other issues.
```suggestion
            active: props.active,
```